app/controllers/content_items_controller.rb

```diff
@@ -136,16 +136,41 @@ def set_guide_draft_access_token
     @content_item.draft_access_token = params[:token]
   end
 
+  class GdsApi::Response
+    def update_response_body
+      updated_response = JSON.parse(@http_response.body).dig("data", "edition").deep_transform_keys(&:underscore)
+      @http_response = RestClient::Response.create(
+        updated_response.to_json,
+        @http_response.net_http_res,
+        @http_response.request,
+      )
+    end
+  end
+
   def load_content_item
     content_item = Services.content_store.content_item(content_item_path)
 
-    content_item["links"]["ordered_related_items"] = ordered_related_items(content_item["links"]) if content_item["links"]
-
-    @content_item = PresenterBuilder.new(
-      content_item,
-      content_item_path,
-      view_context,
-    ).presenter
+    @content_item = if content_item["schema_name"] == "news_article" && (Features.graphql_feature_enabled? || params.include?(:graphql))
```

Comment:
I'd like to see this conditional given a name. The schema_name check might be a separate concept to the combination of feature flag and query param, too.

Reply:
I've moved the feature flag/query string logic into a method caled `serve_response_from_graphql?`.